### PR TITLE
Docs for checking k8s: closes #123

### LINKED ISSUE #123: [docs] check k8s

### PR DESCRIPTION
**Pull request by thuey**
@jonnymbgood I put `kubernetes` as the alias because I think `k8s` should be canonical. It is, however, the abbreviated version, so I could be persuaded to switch them around.

### CODE REVIEW

**Comment by jonnymbgood** [comment]
I kind of think we should use the full name as the canonical version.

File: internal/server/action/checkK8s.go [hunk 1]
```diff
@@ -44,6 +44,9 @@ func checkK8sNamespace(clusterId string, ns string, ctx Context) (response Respo
 
 var checkK8sTarget = &target{
 	id: "k8s",
+	aliases: [][]string{
```
Comment (jonnymbgood) [inline internal/server/action/checkK8s.go]: Lets go ahead and make the id kubernetes and the alias k8s

**Comment by thuey** [comment]
Okay. I can make the change

**Comment by thuey** [comment]
@jonnymbgood I made `kubernetes` canonical and added some more visibility on the `check k8s` commands. Let me know what you think.

**Pull request merged**